src/tests/commonTests.ts

```diff
@@ -1022,9 +1022,9 @@ export async function t9n(
       const props: Partial<MessageBundle> = {};
 
       for (const prop of nonIntlProps) {
-        const index = nonIntlProps.indexOf(prop);
-        props[prop] = `${index}`;
-        component.setProperty(prop, `${index}`);
+        const index = nonIntlProps.indexOf(prop).toString();
```

Comment:
Nice! ✨🧹✨

Alternatively: `${nonIntlProps.indexOf(prop)}`

This is going into a feature branch, so we don't have to split this up into a separate refactor PR. 🎉

Reply:
Can you update the name of nonIntlProps as `extraMessages` and do the same for the method name.

It was supposed to be updated [here ](https://github.com/Esri/calcite-components/pull/5465#discussion_r998674621) and I guess while changing the base branch the changes are wiped off.

Reply:
> Can you update the name of nonIntlProps as additionalMessages and do the same for the method name.

will do!

> This is going into a feature branch, so we don't have to split this up into a separate refactor PR. 

lol I don't like randomly pushing to other people's branches. Franco be like:

![personal-space](https://user-images.githubusercontent.com/10986395/197311143-bf0c9600-8b42-4107-8506-c3650c35e522.gif)